### PR TITLE
#5326: Add tensor manip ops to ttnn

### PR DESCRIPTION
**Pull request by Aswinmcw**
Add tensor manip ops to ttnn
Sweep Test Results:
[full_like.csv](https://github.com/tenstorrent-metal/tt-metal/files/14296286/full_like.csv)
[full.csv](https://github.com/tenstorrent-metal/tt-metal/files/14296287/full.csv)
[maeloss.csv](https://github.com/tenstorrent-metal/tt-metal/files/14296288/maeloss.csv)
[mean_hw.csv](https://github.com/tenstorrent-metal/tt-metal/files/14296290/mean_hw.csv)
[mseloss.csv](https://github.com/tenstorrent-metal/tt-metal/files/14296291/mseloss.csv)
[normalize_global.csv](https://github.com/tenstorrent-metal/tt-metal/files/14296292/normalize_global.csv)
[normalize_hw.csv](https://github.com/tenstorrent-metal/tt-metal/files/14296293/normalize_hw.csv)
[ones.csv](https://github.com/tenstorrent-metal/tt-metal/files/14296294/ones.csv)
[softplus.csv](https://github.com/tenstorrent-metal/tt-metal/files/14296295/softplus.csv)
[std_hw.csv](https://github.com/tenstorrent-metal/tt-metal/files/14296296/std_hw.csv)
[var_hw.csv](https://github.com/tenstorrent-metal/tt-metal/files/14296297/var_hw.csv)
[zeros_like.csv](https://github.com/tenstorrent-metal/tt-metal/files/14296298/zeros_like.csv)
[zeros.csv](https://github.com/tenstorrent-metal/tt-metal/files/14296299/zeros.csv)

### CODE REVIEW

**Comment by Aswinmcw** [comment]
Fast Dispatch Test: https://github.com/tenstorrent-metal/tt-metal/actions/runs/7915488148
Slow Dispatch Test: https://github.com/tenstorrent-metal/tt-metal/actions/runs/7915489277

File: ttnn/ttnn/operations/others.py [hunk 1]
```diff
@@ -505,4 +505,611 @@ def upsample(
     return output_tensor
 
 
+def _zeros_like_validate_input_tensors(operation_name, input_tensor, *args, **kwargs):
```
Comment (arakhmati) [inline ttnn/ttnn/operations/others.py:508]: the newly code should be removed from others.py and moved into creation.py, reduction.py and losses.py

File: tests/ttnn/unit_tests/test_manip.py [hunk 2]
```diff
@@ -0,0 +1,352 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```
Comment (arakhmati) [inline tests/ttnn/unit_tests/test_manip.py:1]: this file should be broken down into test_creation.py, test_reduction.py and test_losses.py. And the files should be under tests/ttnn/unit_tests/operations

**Review by arakhmati — CHANGES_REQUESTED** [review]

**Review by arakhmati — CHANGES_REQUESTED** [review]
I love it! 

But one last thing. Can you please rename `mseloss` to `mse_loss` and `maeloss` to `l1_loss`? We really want to keep our names same as torch

**Review by arakhmati — APPROVED** [review]

**Pull request merged**